data/mods/Aftershock/maps/afs_lab_weapons_range.json

```diff
@@ -1,3 +1,8 @@
+Cataclysm: Dark Days Ahead JSON Web Linting Tool
```

Comment:
Here is the text from the json formatter page.

Reply:
GAH.